tests/masonry/builders/test_editable_builder.py

```diff
@@ -86,6 +86,30 @@ def tmp_venv(tmp_path: Path, env_manager: EnvManager) -> Iterator[VirtualEnv]:
     shutil.rmtree(str(venv.path))
 
 
+@pytest.fixture()
+def bad_scripts_no_colon() -> Poetry:
+    poetry = Factory().create_poetry(
+        Path(__file__).parent.parent.parent
+        / "fixtures"
+        / "bad_scripts_project"
+        / "no_colon"
+    )
+
+    return poetry
+
+
+@pytest.fixture()
+def bad_scripts_too_many_colon() -> Poetry:
+    poetry = Factory().create_poetry(
+        Path(__file__).parent.parent.parent
```

Comment:
There is a `fixture_dir` helper for that. You can find example usage if you search for it in the codebase.